styles/main.css

```diff
@@ -326,96 +235,104 @@ a {
     pointer-events: none;
 } 
 
-.main-title {
+#gallery h1{
     width: 100%;
-    text-align: center;
-    margin: 30px 0;
+    margin: 15px 8px;
 }
 
-#break {
-    width: 580px;
-    height: 5px;
-    margin: auto;
-    background: #cad6cb;
+#internal-page p {
+    margin: 0 0 10px 8px;
+    clear: both;
+}
+
+.main-title {
+    font-size: 18px;
+    font-weight: 100;
+    color: #897F7B;
+}
+
+.sub-title {
+    font-size: 16px;
+    color: #E94E1B;
+    font-family: 'Open Sans';
+    font-weight: 100;
+}
+
+#line {
+    border-top: solid 1px #DEDCDA;
+    margin-bottom: 30px;
 }
 
 .description {
-    width: 580px;
-    line-height: 25px;
+    width: 100%;
+    line-height: 30px;
     margin: 30px auto;
 }
 .source-text, .source-text a {
     font-size: 12px;
     color: #ccc;
 }
 
-#process {
-    max-width: 780px;
-    margin: auto;
-    text-align: center;
+.source-text a:hover {
+    color: #E94E1B;
 }
 
-.img-1col img {
-    max-width: 580px;
-    /*max-height: 400px;*/
-    text-align: center;
-}
 
-.img-1col .suzy {
-    max-width: 780px;
+#process {
+    overflow: hidden;
+    display: block;
+    height: auto;
+    margin-bottom: 40px;
 }
 
-.img-1col .caption {
-    font-size: 13px;
-    line-height: 35px;
-    font-style: italic;
-    padding-bottom: 20px;
-    color: #ccc;
+#process img {
+    float: left;
+    max-width: 310px;
+    margin: 8px;
 }
 
-.img-2col img {
-    max-width: 380px;
-    max-height: 225px;
-    margin: 0 4px;
+#process .designprocess {
+    max-width: 472px;
+    padding: 10px 0;
 }
 
-.img-2col .vertical {
-    max-width: 380px;
-    max-height: 430px;
-    margin: 0 4px;
-}
 
-.img-2col .vertical-s {
-    max-width: 245px;
-    max-height: 250px;
-    margin: 0 6px;
+#process .designprocess-main {
```

Comment:
are you using `.designprocess-main` or `.designprocess` anywhere else besides on your project pages? if not, then why do you need to over specify with `#process`?